.flake8

```diff
@@ -9,7 +9,8 @@ exclude = venv*,__pycache__,node_modules,bower_components
 ignore = D203,W503
 max-complexity = 12
 max-line-length = 120
-putty-ignore =
-    **/__init__.py : +F401
-    app/main/__init__.py, app/status/__init__.py : +E402
-    app/main/__init__.py : +F403
+per-file-ignores =
+    **/__init__.py : F401
+    app/main/__init__.py : E402
```

Comment:
Is this a new ignore here? Why do we need or if it was OK before?

Reply:
It's not a new ignore. The new extension doesn't support multiple files on a single line, so I've just split out the two files onto two lines.

Reply:
You do highlight that I should condense the error codes for main/init though.